src/prefect/core/flow.py

```diff
@@ -1249,6 +1249,10 @@ def deploy(
         if self.storage is None:
             self.storage = get_default_storage_class()(**kwargs)
 
+        if isinstance(self.storage, prefect.environments.storage.Local):
+            self.environment.labels.add("local")
+            self.environment.labels.add(slugify(self.name))
```

Comment:
Can you add a note of these labels being automatically added? Maybe to the docstrings for the Local storage

Reply:
Good call

Reply:
Updated